static/css/cf-enhancements.less

```diff
@@ -804,3 +807,38 @@ tbody {
     name: EOF
     eof: true
 */
+
+
+/* topdoc
+  name: Warning
+  family: cf-forms
+  notes:
+    - "Adds Warning state to form fields"
+  tags:
+    - cf-forms
+*/
+
+@input-warning:     @gold;
```

Comment:
This can be removed, it's already in `cf-theme-overrides.less` https://github.com/cfpb/cfgov-refresh/blob/flapjack/static/css/cf-theme-overrides.less#L125